fairlearn/postprocessing/_threshold_optimizer.py

```diff
@@ -409,11 +526,19 @@ def _threshold_optimization_for_equalized_odds(self, sensitive_features, labels,
         self._x_grid = np.linspace(0, 1, self.grid_size + 1)
         y_values = pd.DataFrame()
 
-        for sensitive_feature_value, group in data_grouped_by_sensitive_feature:
-            roc_convex_hull = _tradeoff_curve(group, sensitive_feature_value, flip=self.flip)
-            self._tradeoff_curve[sensitive_feature_value] = \
-                _interpolate_curve(roc_convex_hull, 'x', 'y', 'operation', self._x_grid)
-            y_values[sensitive_feature_value] = self._tradeoff_curve[sensitive_feature_value]['y']
+        for (
```

Comment:
this particular black rewrite (i presume?) looks a bit awkward to me...

but it's fine with me if it's fine with you... but the next time i touch this code, i might tweak it :-)

Reply:
I'd recommend not tweaking black formatted code (although I agree it sometimes very awkward), so that we can move towards having the code automatically formatted in a git pre-hook.